src/app/music-data/screens/albums-screen.component.tsx

```diff
@@ -1,10 +1,40 @@
-import React from 'react';
-import { View, Text } from 'react-native';
+import React, { useCallback } from 'react';
+import { FlatList } from 'react-native';
+import Icon from 'react-native-vector-icons/dist/Ionicons';
+import { useTheme } from 'styled-components';
+import styled from 'styled-components/native';
+
+import { albums } from '../../../mocks/albums';
+import { Container } from '../../ui/container.component';
+import { AlbumComponent } from '../components/album.component';
+import { MusicDataNavigationProps } from '../route.params';
+
+export const BackButtonContainer = styled.TouchableOpacity`
+    margin-left: ${(props) => props.theme.spacer * 3}px;
+    margin-top: ${(props) => props.theme.spacer * 2}px;
+`;
+
+export type MusicDataProps = MusicDataNavigationProps<'Albums'>;
+
+export const AlbumsScreenComponent: React.FC<MusicDataProps> = (props: MusicDataProps) => {
+    const theme = useTheme();
 
-export const AlbumsScreenComponent: React.FC = () => {
     return (
-        <View>
-            <Text>Albums screen</Text>
-        </View>
+        <Container>
+            <BackButtonContainer
```

Comment:
You really should create back-button-component in /ui folder and use it in all screens